pkgs/os-specific/linux/roccat-tools/default.nix

```diff
@@ -0,0 +1,112 @@
+{ stdenv, fetchurl, cmake, dbus, glib, libcanberra, atk, cairo
+, gdk_pixbuf, gtk2, pango, libnotify, libgudev, xorg, sqlite
+, gegl, dbus_glib, gettext, pkgconfig, shadow, devices ? false
+}:
```

Comment:
Selecting individual devices was never intended to be a user facing feature. More of an addition which reduces compile time and creates less clutter in the desktop menu.
Could you point me towards the documentation for concatStringsSep as I am unable to find any and will struggle to use something I do not understand.
Thanks.